packages/shared/src/components/Feed.tsx

```diff
@@ -436,6 +452,14 @@ export default function Feed<T>({
           onRequestClose={closeSharePost}
         />
       )}
+
+      {lazyModal?.type === LazyModal.CompanionModal && (
+        <ActivateCompanionModal
+          isOpen
+          onRequestClose={() => companionModalToggle(false)}
+          {...lazyModal.props}
+        />
+      )}
```

Comment:
Ah you can actually simple call openModal from lazyModal to open it.
No need to have it explicitly in the files anymore.
This way you can potentially also move the logic to useCompanionTrigger?